packages/fiori/src/Page.js

```diff
@@ -0,0 +1,207 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import PageBackgroundDesign from "./types/PageBackgroundDesign.js";
+
+// Template
+import PageTemplate from "./generated/templates/PageTemplate.lit.js";
+
+// Styles
+import PageCss from "./generated/themes/Page.css.js";
+
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-page",
+	managedSlots: true,
+	languageAware: true,
+	properties: /** @lends sap.ui.webcomponents.fiori.Page.prototype */ {
+
+		/**
+         * This property is used to set the background color of the <code>ui-page</code>. When a list is placed inside the page, the value "List" should be used to display a gray background. "Standard", with the default background color, is used if not specified.
+		 * <br><br>
+		 * Available options are:
+		 * <ul>
+		 * <li><code>List</code></li>
+		 * <li><code>Solid</code></li>
+		 * <li><code>Standard</code></li>
+		 * <li><code>Transparent</code></li>
+		 * <ul>
+		 * @type {PageBackgroundDesign}
+		 * @defaultvalue "Standard"
+		 * @public
+		 */
+		backgroundDesign: {
+			type: String,
+			defaultValue: PageBackgroundDesign.Standard,
+		},
+
+		/**
+         * Disables vertical scrolling of page contents. Page headers and footers are fixed and do not scroll.
+         * If set to true, there will be no vertical scrolling at all.
+         *
+		 * @type {Boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disableScrolling: {
+			type: Boolean,
+		},
+
+		/**
+         * Decides whether the footer can float.
+         * When set to true, the footer is not fixed below the content area anymore, but rather floats over it with a slight offset from the bottom.
+         *
+		 * @type {Boolean}
+		 * @defaultvalue true
+		 * @public
+		 */
+		floatingFooter: {
+			type: Boolean,
+		},
+
+		/**
+         * Decides whether the footer would be visible.
+         *
+		 * @type {Boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		hideFooter: {
```

Comment:
This property is not needed. If someone wants to hide the footer, they can just remove it from the DOM